src/js/selectedcategorymarkup.js

```diff
@@ -0,0 +1,29 @@
+import getBooksByCategory from './service/getBooksByCategory';
+import bookCard from '../temlpates/bookCard.hbs';
+import { bookslist1 } from './querrySelectors';
+import clearTitles from './clearcategorytitles';
+import clearBestSellersMarkup from './clearbsmarkup';
+
+const request = 'Combined Print and E-Book Nonfiction';
+
+const testBtn = document.querySelector('.test-btn');
+const categoryTitle = document.querySelector('.best-sellers-header');
+
+const selectedCategoryMarkup = async () => {
+  clearTitles();
+  clearBestSellersMarkup();
```

Comment:
instead of clearing html of best-sellers section create new section that will be shown when user will try to search books in any category, and this click will hide best-sellers section